doc/user-guides/kernel-parameters.rst

```diff
@@ -207,8 +207,8 @@ relevant for configuring or debugging ACRN-based systems.
        from the guest VM.
 
        If hypervisor relocation is disabled, verify that
-       :option:`hv.MEMORY.HV_RAM_START` and :option:`hv.MEMORY.HV_RAM_SIZE`
-       does not overlap with the hypervisor's reserved buffer space allocated
+       :option:`hv.MEMORY.HV_RAM_START` and the hypervisor RAM size computed by the linker
```

Comment:
@NanlinXie @Kunhui-Li can you verify this is the correct description to replace the previous mention of HV_RAM_SIZE.